modules/apps/commerce/commerce-price-list-test/src/testIntegration/java/com/liferay/commerce/price/list/internal/model/listener/test/CPInstanceUnitOfMeasureModelListenerTest.java

```diff
@@ -10,12 +10,12 @@
 import com.liferay.commerce.price.list.model.CommercePriceEntry;
 import com.liferay.commerce.price.list.model.CommercePriceList;
 import com.liferay.commerce.price.list.service.CommercePriceEntryLocalServiceUtil;
-import com.liferay.commerce.price.list.test.util.CommercePriceEntryTestUtil;
-import com.liferay.commerce.price.list.test.util.CommercePriceListTestUtil;
 import com.liferay.commerce.product.model.CPInstance;
 import com.liferay.commerce.product.model.CPInstanceUnitOfMeasure;
 import com.liferay.commerce.product.service.CPInstanceUnitOfMeasureLocalServiceUtil;
 import com.liferay.commerce.product.test.util.CPTestUtil;
+import com.liferay.commerce.test.util.price.list.CommercePriceEntryTestUtil;
```

Comment:
I did this refactor because the commerce-price-list-test module doesn't export the package (and I think it's not correct to export it because it's not an "util" module). So I moved them in the commerce-test-util like other test util classes (CommerceInventoryTestUtil / CommerceTaxTestUtil / CommerceAccountGroupTestUtil ... )